descartes_moveit/src/moveit_state_adapter.cpp

```diff
@@ -26,6 +26,37 @@
 
 const static int SAMPLE_ITERATIONS = 10;
 
+namespace
+{
+bool getJointVelocityLimits(const moveit::core::RobotState& state,
+                            const std::string& group_name,
+                            std::vector<double>& output)
+{
+  std::vector<double> result;
+
+  auto models = state.getJointModelGroup(group_name)->getActiveJointModels();
+  for (const moveit::core::JointModel* model : models)
+  {
+    const auto& bounds = model->getVariableBounds();
+    // Check to see if there is a single bounds constraint (more might indicate
+    // not revolute joint)
+    if (bounds.size() != 1)
```

Comment:
+1 I did that on the IK seed generator, but wasn't aware of it when I started with this